### DISCUSSION TITLE
Bolded text isn't preserved. What should I do?

### DISCUSSION THREAD

**Discussion by chtx** [akosbalasko/yarle#277] [Q&A]
Hi. I just run the program and the conversion worked as expected apart from the bold text, which disappeared completely. Interestingly, highlights were preserved. 

I've used an enex file generated by Evernote Legacy on Windows. 
I don't see an option to tick to preserve the bold text, so I assume it should do that by default.

What should I do? Thanks. 

EDIT: I've run a few tests on single notes. Some of them preserve the bolded text, some don't.

**Comment by chtx** [comment]
I investigated a bit more. 

The .enex file where the bold text wasn't preserved at all, the tags are as follows:

For bold: 
`<span style="font-weight: bold;"></span>`

For bold AND highlight:
`<span style="background-color: rgb(255, 250, 165); font-weight: bold;-evernote-highlight:true;"></span> ` 

In another note where the bold text was preserved that tags are:
`<b></b>`

But in the same note, what wasn't preserved is bold AND highlight (the output was only bold without highlight). In that case the tags are as follows:
`<b style="background-color: rgb(255, 250, 165);-evernote-highlight:true;"></b>`

But (still in the same note) when highlight was only a part of a longer bold text, then all worked. The tags were different though: 
`<b><span style="background-color: rgb(255, 250, 165);-evernote-highlight:true;">Highlight and bold</span>Continuation of the bold text</b>`

I hope its not too confusing. 

I zipped the 2 files I analyzed and I'm attaching them. 
[Evernote Export.zip](https://github.com/akosbalasko/yarle/files/6816762/Evernote.Export.zip)

Thanks.

**Comment by akosbalasko** [comment]
Thank you for this detailed investigation, I look into it asap.
